config/bruno

```diff
@@ -1,7 +1,12 @@
 include config/dev
+<<<<<<< 69ea292da60cf6a4790cfa85e3478bb599d2c839
```

Comment:
Are you sure you want to merge conflicts in your config file? :P

Reply:
F** another rebasing issue. I should have been more careful, I apologize.